### ISSUE TITLE
Can't edit Definitions.

### ISSUE THREAD

**Issue by IanWONeill** [nshusa/rsam#4]
Less an issue more a hopeful future request, We both know the RS2 private development scene is littered with Customs and Hybrids. Take a look at this repo: 
https://github.com/TRMayhem/RS2-Definition-Suite/

It's in C# but regardless of the language this would be an excellent feature to add to the Asset Manager.
Thoughts?

**Comment by nshusa** [comment]
@IanWONeill 
This would be a plugin if anything to the Asset Manager. This is super easy to do, the only issue and reason I never did this is because no one decodes any config file the same especially for items, npcs, objects. When newer revisions came out people wouldn't format the data into 317 format they just copied and pasted the "readValues" not knowing what the new opcodes did.

As far as this if you can think of a design where you can support all revisions I would do this. Users would most likely have to write up a meta file that basically tells my program how to decode their configs.

**Comment by IanWONeill** [comment]
What I was aiming at with this was to allow the editing of the 'definitions' for instance where people have been forever hard-coding extra actions to objects for instance, an adjustment would let them load it from the cache keeping the Client clean. It could also be a way for importing the OSRS Examines into the 317 format cache instead of having 'It's a Ian' or 'It's a Nshusa' 💯 they could use the lists from:
https://rune-status.net/threads/item-examines-from-rev-154.2396/
https://rune-status.net/threads/npc-examines-from-rev-154.2397/
https://rune-status.net/threads/object-examines-from-rev-154.1968/
Instead of what they (should be) doing and adding extra packets to the 317 protocol to be able to send the data as OSRS does it. I think spoofing the Examines in real-time RS is more efficient that Scraping wikis(?)

So basically to wrap it up: 1. Ability to edit definitions directly in the Cache, 2: Importing Definitions
What do you think to those ideas?

**Comment by nshusa** [comment]
@IanWONeill Really doubt anyone will stick with the default 317 values. Especially as old school comes out with new features they'll want to implement those in the 317 client. I really don't want to waste my time making those tools.

> I think spoofing the Examines in real-time RS is more efficient that Scraping wikis(?)

Nah, if you parse the wiki page in parallel you can grab all the information off wiki for 20k+ items in less than 10 seconds. That information hardly ever changes but the next time you parse the page check for any changes if there's a change then update your data if not then skip. That's what I do for my tool its super fast.
